src/ascii_reader.rs

```diff
@@ -54,7 +56,11 @@ impl<'a> AsciiStlReader<'a> {
                     "empty file?",
                 ))
             }
-            _ => {}
+            Some(Ok(ref line)) => {
+                if line.trim().len() > 5 {
+                    name = Some((line["solid".len()..].trim()).to_string());
```

Comment:
Looks like this should use `.strip_prefix`: https://doc.rust-lang.org/std/primitive.str.html#method.strip_prefix

Reply:
Yea good catch